packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -130,13 +130,12 @@ const Home = () => {
                     {heroFeatures.map((feature, index) => {
                         const featureMessage = feature.message
                         return (
-                            <Box
-                                key={index}
-                                background={'white'}
-                                boxShadow={'0px 2px 2px rgba(0, 0, 0, 0.1)'}
-                                borderRadius={'4px'}
-                            >
-                                <Link target="_blank" href={feature.href}>
+                            <Link key={index} target="_blank" href={feature.href}>
+                                <Box
+                                    background={'white'}
+                                    boxShadow="0px 2px 2px rgba(0, 0, 0, 0.1)"
+                                    borderRadius={'4px'}
```

Comment:
Sorry, should have made this clear for all lines here. I thought you would have noticed. 😅
```suggestion
                                   background='white'
                                    borderRadius='4px'
```